microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-properties.json

```diff
@@ -25,5 +25,19 @@
 		"context": {
 			"dependency": "quarkus-jaeger"
 		}
+	},
+	"Add Container Image properties": {
+		"prefix": "cip",
+		"body": [
+			"quarkus.container-image.group=$1",
+			"quarkus.container-image.registry=${2|docker.io,quay.io|}",
+			"quarkus.kubernetes.expose=${3|true,false|}",
+			"#quarkus.container-image.username=$4",
+			"#quarkus.container-image.password=$5"
+		],
+		"description": "Configure Container Image properties",
+		"context": {
+			"dependency": "quarkus-container-image"
```

Comment:
So this requirement is not sufficient. Users can only add quarkus-container-image-docker (not sure whether the container properties are relevant to the quarkus-container-image-jib and quarkus-container-image-s2i extensions). But the properties are not even proposed until quarkus-container-image-deployment.jar is added to the classpath (so here there's a discrepancy between the extension name and the jar)

Long story short, there's a missing link here to properly support container properties. This needs to be addressed in a new ticket.

In the meantime, I'd remove this snippet, as it's pretty much unusable until the above issue is fixed.

Reply:
Thank you for looking into that, fixed